great_expectations/data_context/store/__init__.py

```diff
@@ -36,11 +37,13 @@
     ".html_site_store",
     ".profiler_store",
     ".metric_store",
+    ".datasource_store",
```

Comment:
Missed this in my previous store implementation 😬

Reply:
All good - nice job catching it. Is there a good test to add here so we catch issues like this in the future? Just off the cuff, there might be a better way: maybe a test for inclusion of all modules with classes that inherit from `Store`?